aspnetcore/security/authorization/roles.md

```diff
@@ -59,8 +59,8 @@ Access to an action can be limited by applying additional role authorization att
 
 In the preceding `ControlAllPanelController` controller:
 
-* Members of the `Administrator` role or the `PowerUser` role can access the controller and the `ShutDown` action.
-* Only members of the `Administrator` role can access the `SetTime` action.
+* Members of the `Administrator` role or the `PowerUser` role can access the controller and the `SetTime` action.
+* Only members of the `Administrator` role can access the `ShutDown` action.
```

Comment:
I added a screenshot of the current docs page (https://learn.microsoft.com/en-us/aspnet/core/security/authorization/roles?view=aspnetcore-9.0#adding-role-checks), because this might be easier to review.

![image](https://github.com/user-attachments/assets/acc4ee7a-6e0b-4ea9-8fb5-530df038a1c9)